.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "lib/data_cleanup/rules"]
 	path = lib/data_cleanup/rules
 	url = https://github.com/DigitalCurationCentre/data_cleanup_rules.git
+	fetchRecurseSubmodules = false
```

Comment:
This update is needed due to ArgoCD and how it handles private submodules. Hopefully this shouldn't be an issue? There are alternative workarounds but this is the recommendation from the team. [link](https://github.com/argoproj/argo-cd/issues/10174#issuecomment-1209816345)

> submodule.<name>.fetchRecurseSubmodules
> This option can be used to control recursive fetching of this submodule. If this option is also present in the submodule’s entry in .git/config of the superproject, the setting there will override the one found in .gitmodules. Both settings can be overridden on the command line by using the --[no-]recurse-submodules option to git fetch and git pull.

 [ref](https://git-scm.com/docs/gitmodules#Documentation/gitmodules.txt-submoduleltnamegtfetchRecurseSubmodules)